This patch is applied on top of the other keyboard patches and is dependant
on them.

Also, swapping your capslock and ctrl keys is something that your Operating
System can do, so patching the EC firmware is quite an overkill for this
task.

To enable this patch, you will need to edit the PATCHES-* variable inside
the Makefile

--- x230.G2HT35WW.img.dvorak.hex	2017-09-20 19:57:42.206883067 +0300
+++ x230.G2HT35WW.img.prepatched.hex	2017-09-20 19:58:00.471897920 +0300
@@ -8515,7 +8515,7 @@
 000218b0  9e 9e 00 9f 9f 00 4a 4a  00 00 00 00 b2 00 00 00  |......JJ........|
 000218c0  b4 16 02 00 40 00 00 00  58 18 02 00 18 18 02 00  |....@...X.......|
 000218d0  0b 00 00 00 98 18 02 00  01 02 11 10 1f 6e 2e 83  |.............n..|
-000218e0  00 00 00 00 00 00 00 00  70 03 12 1e 20 2d 2f 00  |........p... -/.|
+000218e0  00 00 00 00 00 00 00 00  70 03 12 3a 20 2d 2f 00  |........p..: -/.|
 000218f0  00 00 00 00 00 00 00 00  71 04 13 72 21 73 30 00  |........q..r!s0.|
 00021900  00 00 00 00 00 00 00 00  06 05 14 15 22 23 31 32  |............"#12|
 00021910  00 00 00 00 00 00 00 00  07 08 17 16 25 24 34 33  |............%$43|
@@ -8529,8 +8529,8 @@
 00021990  00 00 00 00 00 00 00 00  50 51 00 00 00 53 7e 4f  |........PQ...S~O|
 000219a0  00 00 00 00 00 00 00 00  00 7c 7d 00 00 3c 00 3e  |.........|}..<.>|
 000219b0  00 00 00 00 00 00 00 00  00 00 00 2c 00 00 39 00  |...........,..9.|
-000219c0  00 00 00 00 00 00 00 00  3a 00 00 00 00 00 40 00  |........:.....@.|
-000219d0  00 00 00 00 00 00 00 96  00 9d 00 9e 9f 4a 3a 9c  |.............J:.|
+000219c0  00 00 00 00 00 00 00 00  1e 00 00 00 00 00 40 00  |..............@.|
+000219d0  00 00 00 00 00 00 00 96  00 9d 00 9e 9f 4a 1e 9c  |.............J..|
 000219e0  7d 7e 00 00 00 00 00 00  ff 00 7f 00 7f 00 ff 00  |}~..............|
 000219f0  ff 00 ff 00 ff 00 ff 00  ff 00 8b 00 ff 00 d3 00  |................|
 00021a00  e3 00 a6 00 48 00 41 80  fa 03 00 00 a0 97 00 00  |....H.A.........|
